aws/acm-certificate/outputs.tf

```diff
@@ -1,21 +1,21 @@
 output "certificate_arn" {
   description = "The ARN of the certificate"
-  value       = element(concat(aws_acm_certificate.cert.*.arn, [""]), 0)
+  value       = aws_acm_certificate.cert[0].arn
 }
 
 output "acm_certificate_domain_validation_options" {
   description = "A list of attributes to feed into other resources to complete certificate validation. Can have more than one element, e.g. if SANs are defined. Only set if DNS-validation was used."
-  value       = flatten(aws_acm_certificate.cert.*.domain_validation_options)
+  value       = aws_acm_certificate.cert[0].domain_validation_options
 }
 
 output "acm_certificate_validation_emails" {
   description = "A list of addresses that received a validation E-Mail. Only set if EMAIL-validation was used."
-  value       = flatten(aws_acm_certificate.cert.*.validation_emails)
+  value       = aws_acm_certificate.cert[0].validation_emails
 }
 
 output "validation_route53_record_fqdns" {
   description = "List of FQDNs built using the zone domain and name."
-  value       = aws_route53_record.validation.*.fqdn
+  value       = aws_route53_record.validation[0].fqdn
```

Comment:
Why is this the same?

Reply:
I should actually wrap a coalescelist() around it in case, the count is 0, and no list is created.